src/main/kotlin/coverosR3z/timerecording/types/SubmittedPeriod.kt

```diff
@@ -46,7 +46,7 @@ data class SubmittedPeriod(val id: SubmissionId, val employeeId: EmployeeId, val
             return deserialize(str, Companion) { entries ->
                 try {
                     val id = SubmissionId.make(entries[Keys.ID])
-                    val employeeId = EmployeeId.make(entries[Keys.ID])
+                    val employeeId = EmployeeId.make(entries[Keys.EMPLOYEE_ID])
```

Comment:
This was fixing a bug

Reply:
Yup, a whole new pattern.  UI tests now check afterwards that after restarting, their original database matches the restored database.  That's as future-proof as I know how to go.